core/src/main/java/com/rim/IceField/PlayerBaseGUI.java

```diff
@@ -12,7 +14,7 @@
 //TODO need to highlight the iceberg
 // that is selected by the Polar Explorer
 
-public class PlayerBaseGUI {
+public class PlayerBaseGUI implements InputProcessor {
```

Comment:
this class is for displaying the playerBase image, and is bad practice to combine it with the input processor which is part of the Controller and is part of other game logic, let's keep this separated